### PR TITLE
Finish the Project

### PR DESCRIPTION
**Pull request by jheart-vic**
I created this using Html, CSS, JavaScript, and Webpack
I added a function that allows users to either
Add task
Delete task and 
Edit task

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,8 +2,31 @@ import './style.css';
 import Task from '../modules/task.js';
 
 const task = new Task();
-task.addList('Dance with Sanja', true, 0);
-task.addList('Dance with Toma', false, 1);
-task.addList('Dance with Waleed', true);
-task.addList('Dance with max', true, 3);
-// task.addList({description: 'Dance with Sanja', completed: true, index: 1});
+window.addEventListener('load', task.showLists());
```
Comment (BenMukebo) [inline src/index.js:5]: - [ ] Great job so far 👏🏻 however In this project, you should implement the CRUD (create, update, delete) methods Fistly you should have a new JavaScript file that will include: 
  -  A function for editing task descriptions make sure you implement it. 🥲 
  -  A function for deleting a task (remove an element from the array). (Note: Youcan also implement the deleting feature like [this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/sneak_peek.md#:~:text=Your%20task%20is%20to%20build%20a%20website%20that%20displays%20a%20list%20that%20looks%20and%20behaves%20like%20the%20part%20of%20minimalist%20project%20capture%20in%20the%20video%20below.)).🥲 
  - Make sure new tasks have the property `completed` set to `false`  by default and the property `index` set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5).🥲 
  - Don't forget, Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.🥲 
  - And make sure, All changes to the To-Do List is saved in local storage. 🥲 
 

- [ ] Kindly implement all those new functionalities currently I can only create the book, I hope this is more clear now👍🏻 . If you have any questions please do not hesitate to tag me on @BenMukebo or send me a direct message on slack✌🏻

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @jheart-vic,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- Your Pull Request has a proper title and description ✔️   
- No linter errors. ✔️ 
- GitHub flow was correctly used.  ✔️   
- A function for adding a new task is implemented.💯 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


- [ ] Please, add a meaningful  `Project description`. Remember A project description is a high-level overview of why you're doing a project. The document explains a project's objectives and essential qualities. Is an essential document that provides your audience with a comprehensive overview of the project. Think of it as the elevator pitch that focuses on what and why without delving into how.

- [ ] Kindly, it would be nice to add explicit steps for runing your app, as this will be of help to non-technical users. You can put the steps to clone and run the project in a new Getting Started section under the project's description. This includes `npm install` , `npm start`, etc.
- [ ] I would also recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```

Please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.
- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (jheart-vic) [reply src/index.js:5]: I already Implemented them

Comment (BenMukebo) [reply src/index.js:5]: Okay let me check again

**Review by Meltrust — APPROVED** [review]
Hi @jheart-vic,


## Wow, you did it 🎉
Brilliant
## Thank you for implementing the changes requested by the previous reviewer 💪 🥇 ㊗️ 
### Well done!




Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### To highlight

- Thank you for improving your readme, as requested by the previous reviewer✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**